docs/compute-element/hosted-ce.md

```diff
@@ -0,0 +1,110 @@
+OSG Hosted CE
+=============
+
+
+## Overview
+
+As an alternative to setting up your own CE, OSG offers a hosted CE option
+to connect your existing campus HPC/HTC cluster to the Open Science Grid.
+The OSG team will host and operate the HTCondor compute element which routes user jobs to your cluster,
+configured for the science communities that you choose to support.
+
+![managed services diagram](/compute-element/img/screenshot_2983.png)
```

Comment:
We keep images in `docs/img/` but I think I like this better. However, can we get more descriptive names for these two images?

Reply:
You mean for the file path?

Reply:
Yup, sorry for the vagueness